src/routes/docs/products/auth/security/+page.markdoc

```diff
@@ -45,6 +45,11 @@ Password dictionary protects users from using bad passwords. It compares the use
 
 Password dictionary can be enabled in the Auth service's **Security** tab on the Appwrite Console.
 
+# Password encryption  {% #password-encryption %}
```

Comment:
I think we might do encryption as well, but the title here should be `Password hashing`